test/test_core.py

```diff
@@ -5830,6 +5830,8 @@ def test_stat_mknod(self):
 
   @also_with_wasmfs
   def test_fcntl(self):
+    if self.get_setting('WASMFS'):
+      self.emcc_args += ['-sFORCE_FILESYSTEM']
```

Comment:
Why is `-sFORCE_FILESYSTEM` needed for WASMFS but not for the old FS?

Reply:
The old FS happens to get included automatically in more cases. Imagine that you do `fopen()`, then in the JS FS that means the syscall does `FS.open()`, which forces the `FS` object to be added. But that's just because the JS FS is implemented in JS. With WasmFS the syscall and support code is all in wasm, and no JS is included because of it. So there is less of a chance of getting the code you want to call manually from JS "by luck".